azurerm/internal/services/network/parse/firewall_policy.go

```diff
@@ -31,3 +32,28 @@ func FirewallPolicyID(input string) (*FirewallPolicyId, error) {
 
 	return &policy, nil
 }
+
+type FirewallPolicyRuleCollectionGroupId struct {
+	ResourceGroup string
+	PolicyName    string
+	Name          string
+}
+
+func FirewallPolicyRuleCollectionGroupID(input string) (*FirewallPolicyRuleCollectionGroupId, error) {
```

Comment:
Could you make this function use `azure.ParseAzureResourceID` rather than depending on `FirewallPolicyID` and a regex match?

Reply:
sure